automation/taskcluster/lib/tasks.py

```diff
@@ -289,11 +336,11 @@ def _craft_build_ish_task(
             features['chainOfTrust'] = True
         if any(scope.startswith('secrets:') for scope in scopes):
             features['taskclusterProxy'] = True
-
         payload = {
             "features": features,
+            "env": env_vars,
             "maxRunTime": 7200,
-            "image": "mozillamobile/fenix:1.3",
+            "image": "mozillamobile/fenix:1.4",
```

Comment:
Did you rebuild the image? (Since this PR doesn't contain any `Dockerfile` changes).

Reply:
@npark-mozilla published a 1.4

Reply:
That's odd, I thought I already updated to 1.4 when I pushed 1.4 docker image